### ISSUE TITLE
Seemingly random issue with "Set Folder Permissions" Again

### ISSUE THREAD

**Issue by morgenweck** [nvisionative/nvQuickSite#110]
I tried using the installer tonight for the first time and I seem to have hit a wall.  I am constantly getting the error message that is on your other issue that is closed.

I tried to install 8.0.3

![image](https://cloud.githubusercontent.com/assets/19788905/15843772/bb210a32-2c32-11e6-9221-452a2b5268f0.png)

The only thing that is different than many of the installs is that my SQL server is on a different box.  All of my DNNs use a two server system.  I am more than willing to work with you to find out the cause of the problem. 

The concept is great but it does not work for me yet.

Bill
Medical University of South Carolina

**Comment by morgenweck** [comment]
Additional information:  I have a Web server ( IIS is 7.5)  that is a Windows 2008 R2 Enterprise and a MS SQL 2008 R2 server running also on a Windows 2008 R2 Enterprise.  Both machines are virtual boxes on the same internal network and this type of set up has been running for 5 years.  Before that it was metal boxes.

**Comment by leigh-syndicateplus** [comment]
I get the same issue.

**Comment by hismightiness** [comment]
@leigh-pointer First, it's great to see you participating here.  Welcome!  

Is your SQL server also on another server like Bill?

**Comment by nvisionative** [comment]
nvQuickSite stores the SQL Server database and transaction logs on the site installation path (**Database** folder).  In order to set the appropriate folder permissions, it attempts to give access to the **NT Service\MSSQLSERVER** account.  For those that have their SQL Server instance on a separate machine, this account will obviously not exist.  Thus the error.  @morgenweck @leigh-pointer @hismightiness - do you guys have any ideas on how to better handle this?  If so, we are willing to implement and test.  Thanks!

**Comment by JeanCollas** [comment]
Same here... 
SQL Express uses this account:
NT Service\MSSQL$SQLEXPRESS

**Comment by nvisionative** [comment]
Thank you @JeanCollas - we will look into better handling for SQL Server Express.

**Comment by rlriggs** [comment]
First of all, thank you for this incredible project - saves me so much time.  For anyone searching the thread, I had this 'Some or all identity references...' error on a new laptop.   Tried many different things.  In my case, turned out not all IIS components were installed.  I compared the IIS components (Add/Remove Windows Features) on my desktop and matched things up, and everything works great on the laptop as well.
